[style, article]: Fix two-column layout and add quantum consciousness article (#10)
Greybeard-Entertainment/pages: docs/quantum-consciousness.org

We all know what we mean when a clone in science fiction would say is this the "*you* - you, or the other - you".  Unfrtounately, that is where the clarity of the problem ends.  Some take it into the mystical domain, arguing that consciousness is your soul.  Some take it into the opposite, and equally stupid direction: you have no consciousness, it's all an elaborate illusion.  What I find most painful, is that while the former camp is clearly labelled as a bunch of crackpots that should not be taken seriously, the second camp[fn:2], is considered the /rational point of view/.  The truth of the matter is that we don't know.  Anyone who claims otherwise, have chosen to ignore more than half of the problem, and have thus come up with a meaningless answer.

> With that said, there is a surprising resurgence in interest related to the question of what does it mean to understand somehing.  What does it mean to know something.  Is this connected to consciousness, or something that our bodies can do autonomously.  Chomsky's room is a good example of this resurgence, despite the mental experiment being proposed a good few years back, transformer-based next-token prediction machines that show a clear propensity to mimic intelligent discourse have sparked a resurgence in that discussion.  Can machines think?  If machines can think now, can a future machine be built that can?  Does the fact that we are built out of meat and machines are built out of something else matter for the ability to understand and synthesise cognition?  Is symbolic reasoning even necessary to call something conscious?

Reviewer: > Is symbolic reasoning even necessary to call something conscious?

That's a great topic to raise because "symbolic reasoning" goes partially towards the "memorizing" part: we're not just remembering facts; we process said facts and find some links, for instance, **causal**, **chronological**, **spatial**, **thematic**, etc. So, "symbolic reasoning" is not just about using symbols, not just about learning ones from others, it's all about introducing new abstract symbols and memorizing them with a given time interval.

Plus, **while neurons work as detectors**, the transformer models we interact with so far **display a curious property**: I don't remember them **requesting clarifications** when I ask whatever questions that come to mind, however ridiculous they are. Even if this architecture displays symbolic reasoning, it's all about the levels of abstraction and the relationships between symbols, as "[Scaling Monosemanticity: Extracting Interpretable Features from Claude 3 Sonnet](https://transformer-circuits.pub/2024/scaling-monosemanticity/)" easily demonstrates.

Now, just to confirm my point, let's conduct a silly test: I'm using a Bluetooth mouse, so it'll be a question about Harald Bluetooth. And I love otters, so let's ask [GPT4-o](https://chatgpt.com/share/2ba7e3a3-8d33-4847-a93f-d373d4f271e8) about king's pet otter to validate whether it's checking its confidence about symbols or not!

> According to legend, Harald Bluetooth had a pet otter named "Sot" (sometimes spelled "Sote").

> Sot was not an ordinary otter but was considered to have almost mystical or supernatural qualities.

> The legend suggests that Harald was very fond of Sot and the otter was often seen by his side.

> There is no concrete historical evidence to confirm the existence of Harald Bluetooth’s pet otter. The story is mostly passed down through oral traditions and later medieval texts.

> Harald Bluetooth, a king of Denmark and Norway in the 10th century, is a significant historical figure known for his role in unifying Denmark and converting the Danes to Christianity. However, the story of Harald Bluetooth's pet otter is more of a myth or legend rather than a well-documented historical fact.

Similarly, atomicity seems to be largely traditional.  Sure, /you/ might have the consciousness of a human, but that, in and of itself is a connected causal system with more than one instance.  To state that two identical copies of your body /cannot/ have a consciousness is preposterous, because you are a collection of small entities -- cells working in tandem, but not always.  Perhaps higher forms of organised matter have higher forms of consciousness, and lower forms too.  By that moniker, however, reduction to the absurd would suggest that something like an electron might have a consciousness[fn:16].  But there are no /good reasons/ to believe that /that/ is not the case.

> What is ifinitely more frustrating is that qualia seem to prevent us from doing any form of experiment to clarify if these notions are accidental or a pattern.  I can foresee an experiment in which I, construct a near-identical clone of myself, and transplant one neuron at a time into that clone.  From a qualia perspective I should be able to exactly pinpoint the moment when my concsiousness stopped, and /if/ it resumes, the moment that it did resume.  It is possible to iterate this experiment (in principle), to establish whether or not these notions are justified.  But the main problem is that the third principle is still in effect: I would come to the same conclusion as before, if I were a zombie.  The information that was obtained is itself a qualia, and is indistinguishable from a similar experiment with no consciousness involved!

Reviewer: I love the idea, as you put semantics versus calculation, and calculation shows a bunch of very different processes. I'd love to return to this discussion, as this is something we don't usually do regarding many terms and dig ourselves massive pits with semantics on topics like **intelligence**: there's a lot about separate mechanisms, timings, information structures, all put in a single poor word.

Couple typo corrections, though.

```suggestion
What is infinitely more frustrating is that qualia seem to prevent us from doing any form of experiment to clarify if these notions are accidental or a pattern.  I can foresee an experiment in which I, construct a near-identical clone of myself, and transplant one neuron at a time into that clone.  From a qualia perspective I should be able to exactly pinpoint the moment when my consciousness stopped, and /if/ it resumes, the moment that it did resume.  It is possible to iterate this experiment (in principle), to establish whether or not these notions are justified.  But the main problem is that the third principle is still in effect: I would come to the same conclusion as before, if I were a zombie.  The information that was obtained is itself a qualia, and is indistinguishable from a similar experiment with no consciousness involved!
```

** Beyond philosophical censorship

In Physics, similar conspiracies were often an indication that our previous models were inadequate; we couldn't just alter our knowledge, we needed to change what we operated on.

One way is like Dirac, to admit defeat: to state that consciousness is fundamentally not intelligible from the point of view of science, and that it can only be interrogated as something that manifestly is part of our physical reality, that it can effect change in the physical reality, but not /of physical reality/, there are no positive physical laws that would dictate how a consciousness operates.

This approach is fine.  I suppose it is not satisfying but not counter-productive either.  This also gives a natural exception to free will, so we can immediately chalk that question up to whatever branch of philosophy is going to interrogate consciousness.  In a perverse way it is a status quo for the subject, unproductive though it is.

> The second approach we might take, is to build a theory around the fact that this information is fundamental.  This appears to be a sound basis, but if not careful devolves into absolute solipsism.  Indeed, consider that we impose as a fundamental restriction that this information must either be universally attainable, or universally unattainable.  One possibility is that we do not exist: universally unattainable, and thus nothing else does.  We are all philosophical zombies.  Obviously, no objective counter-argument can be made, because any argument you would make against that can be traced to "natural roots".  Yet, /you/ know that you exist as a qualia.  The exact opposite of this principle is the basis of "I think, therefore I am".  The qualia of having thoughts and integrated information is a fact.  A subjective one, possibly, but a fact nonetheless.

Reviewer: So "existing as a qualia" is a proper way to discuss this term? It's not about separate senses and the experience, but "the whole bunch of senses poured into a subjective experience"?

Anyway, another funny part here is that we're never talking about "partial philosophical zombies". IIRC, the brain needs a lot of power to do its thing. And maybe some power-saving is involved not only when we sleep. If it's not activating when we're around other people or think of surveillance, but things are mundane and repetitive, no one checks for the lack of qualia. What if it activates at some small intervals all the time? There are many options to put qualia into numbers we're avoiding.

Despite apparent terminological incorrectness, I'll allow myself a silly joke: "Did a zombie eat a hiker's brain in the forest if there are no witnesses?" 

```suggestion
The second approach we might take, is to build a theory around the fact that this information is fundamental.  This appears to be a sound basis, but if not careful, it devolves into absolute solipsism.  Indeed, consider that we impose as a fundamental restriction that this information must either be universally attainable, or universally unattainable.  One possibility is that we do not exist: universally unattainable, and thus nothing else does.  We are all philosophical zombies.  Obviously, no objective counter-argument can be made, because any argument you would make against that can be traced to "natural roots".  Yet, /you/ know that you exist as a qualia.  The exact opposite of this principle is the basis of "I think, therefore I am".  The qualia of having thoughts and integrated information is a fact.  A subjective one, possibly, but a fact nonetheless.
```

The fact of the matter is, Penrose landed in hot water because of a different, and I'd argue somewhat misplaced assertion.  When during a polite shouting match at the canteen in Cambridge I challenged the student that called bullshit on Penrose's entire book, the fact that quantum effects affect only a small part of the physiological processes in the cells, and to a small extent was used as a mainline argument.  And my stance on this question is a bit complex.

** The extent of what we call Quantum

The unfortunate problem that one must deal with, in any discussion that involves the quantum theory, is that huge chunks of it are known very well, others are considered tentatively accepted until a better alternative presents itself, and far too many pieces of it can be chalked up to "we simply don't know".

> Yet another observation that eludes most participants in discussions of Quantum theory, is that it is not /just/, in fact, as some would have you believem, /a theory of physics/, but rather a different model for probability. Complex numbers were finally[fn:4] being accepted into mainstream mathematics, despite being incredibly useful tools for many centuries beforehand, only when the predictions of quantum theory started to become apparent in everyday phenomena.

Reviewer: ```suggestion
Yet another observation that eludes most participants in discussions of Quantum theory, is that it is not /just/, in fact, as some would have you believe, /a theory of physics/, but rather a different model for probability. Complex numbers were finally[fn:4] being accepted into mainstream mathematics, despite being incredibly useful tools for many centuries beforehand, only when the predictions of quantum theory started to become apparent in everyday phenomena.
```
